### PR TITLE
[TT-5866] Update x509-client-auth.md

### PR DESCRIPTION
**Pull request by andyo-tyk**
Fixed a typo and improved wording.

No technical changes, this can be published immediately - not tied to a release.

Prompted by TT-5866

### CODE REVIEW

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md [hunk 1]
```diff
@@ -77,9 +75,9 @@ Your `tyk_analytics.conf` should include these fields at the root level:
 
 
 ## Tyk Pump
-There are 3 mongo pumps, `mongo`, `mongo_aggregate`, and `mongo_selective`.  
+You must separately configure each Tyk Pump that you deploy that targets MongoDB; note that Tyk offers three different Mongo pumps, `mongo`, `mongo_aggregate`, and `mongo_selective`.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]: Not so clear to read. Also 2 `that`

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md [hunk 2]
```diff
@@ -77,9 +75,9 @@ Your `tyk_analytics.conf` should include these fields at the root level:
 
 
 ## Tyk Pump
-There are 3 mongo pumps, `mongo`, `mongo_aggregate`, and `mongo_selective`.  
+You must separately configure each Tyk Pump that you deploy that targets MongoDB; note that Tyk offers three different Mongo pumps, `mongo`, `mongo_aggregate`, and `mongo_selective`.  
 
-In order to setup X509 certificate authentication with MongoDB, you can add the following tags to the `meta` section to each of these 3 pumps, ie:
+The following tags must be added to the `meta` section of each pump:
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]: Again, not so clear to read.

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md [hunk 3]
```diff
@@ -77,9 +75,9 @@ Your `tyk_analytics.conf` should include these fields at the root level:
...
 
 ```json
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]:
```suggestion
```yaml
```

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md [hunk 4]
```diff
@@ -8,19 +8,18 @@ menu:
 weight: 2
 ---
 
-
-You can use the MongoDB X509 Certificate flow to authenticate the Tyk Dashboard, Tyk Pump, and Tyk MDCB with your MongoDB install.  This is slightly different to [AWS DocumentDB setup instructions]({{< ref "frequently-asked-questions/how-to-connect-to-documentdb" >}}).
+You can use the MongoDB X509 Certificate flow to authenticate the Tyk Dashboard, Tyk Pump, and Tyk MDCB with your MongoDB install.  This is slightly different from [AWS DocumentDB setup instructions]({{< ref "frequently-asked-questions/how-to-connect-to-documentdb" >}}).
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]:
```suggestion
You can use the *MongoDB X509 Certificate* flow to authenticate the *Tyk Dashboard*, *Tyk Pump*, and *Tyk MDCB* with your *MongoDB* install.  This is slightly different from [AWS DocumentDB setup instructions]({{< ref "frequently-asked-questions/how-to-connect-to-documentdb" >}}).
```

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md (hunk 2 above)
Comment (letzya) [inline tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]:
```suggestion
The following fields must be set under the `meta` section of each pump (or set as environment variable):
```

File: tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]: How about this?

```suggestion
Tyk offers three different MongoDB pumps (`mongo`, `mongo_aggregate`, and `mongo_selective`) each of which must be separately configured for X509 certificate authentication. 
```

Comment (dcs3spp) [reply tyk-docs/content/tyk-stack/dependencies/mongodb/x509-client-auth.md]:
```suggestion
Tyk offers three different MongoDB pumps (`mongo`, `mongo_aggregate`, and `mongo_selective`), each of which must be separately configured for X509 certificate authentication. 
```

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**